server.js

```diff
@@ -1,24 +1,13 @@
 var artsyEigenWebAssociation = require('artsy-eigen-web-association');
 var express = require('express');
-var https = require('https');
 var http = require('http');
```

Comment:
We might want to force connections to use `https` here - this is how we implemented in kaws: https://github.com/artsy/kaws/pull/149/files

Reply:
The app already redirects to `https://www...` in all cases (other than the apple site-association file, which is intentional).